.github/workflows/Dockerfile.buster

```diff
@@ -17,12 +19,11 @@ RUN apt-get update && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
       git cmake-data=3.18* cmake=3.18* make g++ gperf netcat \
-      python3-minimal python3-dev libpython3-dev python3-jsonschema python3-setuptools python3-pip && \
-    pip3 install wheel && \
+      python3-minimal python3-dev libpython3-dev python3-jsonschema python3-setuptools python3-pip mysql-server libmysqlclient-dev && \
```

Comment:
`jsonschema` is the same package as the others, and it too must have an explicit version...

Reply:
Done.